src/components/Board/Board.component.js

```diff
@@ -298,6 +378,75 @@ export class Board extends Component {
             'is-locked': this.props.isLocked
           })}
         >
+          {isLocked && (
+            <Joyride
+              callback={data => {
+                const { status } = data;
+                if ([STATUS.FINISHED, STATUS.SKIPPED].includes(status)) {
+                  this.setState({ runLiveUnlock: false });
+                }
+              }}
+              steps={firststep}
+              continuous={true}
+              showSkipButton={true}
+              showProgress={true}
+              run={runLiveUnlock}
+              styles={{
+                options: {
+                  // modal arrow and background color
```

Comment:
These comments should be deleted, we want to keep the code as lean as we can, most of the properties are self explanatory, also usually when people change code they forget to update the comment.. so can be a maintenance issue in the future.

Reply:
You are right! They are deleted now!